website/content/docs/configuration/client.mdx

```diff
@@ -201,6 +201,13 @@ client {
 - `host_volume` <code>([host_volume](#host_volume-block): nil)</code> - Exposes
   paths from the host as volumes that can be mounted into jobs.
 
+- `host_volume_plugin_dir` `(string: "")` - Specifies the directory to find host
+  volume plugins. When this parameter is empty, Nomad generates the path
+  using the [top-level `data_dir`][top_level_data_dir] suffixed with
+  `host_volume_plugins`, like `"/opt/nomad/host_volume_plugins"`. This must be
+  an absolute path. Nomad creates the directory on the host, if it does not
+  exist when the agent process starts.
```

Comment:
```suggestion
  an absolute path.
```

it does not get auto-created.

and fwiw, this is the only mention of it in agent logs, whether this param set to something custom or not:

```
[DEBUG] client.fingerprint_mgr.host_volume_plugins: plugin dir does not exist: dir=/opt/nomad/data/host_volume_plugins
```

Reply:
Ah, good catch. I was thinking of the various datadirs which the agent does create, but we never create plugin dirs.